LondonBritishLibrary/orient/BLorient12996.xml

```diff
@@ -0,0 +1,289 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient12996" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Amharic Grammar; Arabic Grammar</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 12996</idno>
+                        <altIdentifier><idno>Or. 12996</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 62</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents><summary/></msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <extent>
+                                    <measure unit="leaf">59+1</measure>
```

Comment:
I think after "leaf" you need  quantity="60".

Reply:
I assume, that Strelcyn considers it the pagination of the anonymous author. He uses the word "original" also for the title. I will try to rephrase it. 

<img width="434" alt="Screenshot 2024_grammar-original-pagination" src="https://github.com/user-attachments/assets/bd764934-22e0-465f-84e9-12ab1203b7a7">